apps/design/backend/src/app.test.ts

```diff
@@ -3637,3 +3639,177 @@ test('decryptCvrBallotAuditIds', async () => {
     expect(contents.CVR[0]).toEqual(cvr.castVoteRecord);
   }
 });
+
+test('getExportedElectionDefinition returns the exported election including reordering updates on render', async () => {
```

Comment:
I think this would make a bit more sense to live in `store.test.ts` because it's not testing the app API. That being said, if it's too much trouble to have it live there, I'm ok with it being here, but maybe we can just change the test title to `store.getExportedElectionDefinition ...` because I was a bit confused at first when I saw it